src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -202,7 +202,7 @@ public virtual bool RunInMemory
         public Size? TransactionSizeLimit { get; protected set; }
 
         [Description("Transaction size limit, for encrypted database only, after which an index will stop and complete the current batch")]
-        [DefaultValue(64)]
+        [DefaultValue(96)]
```

Comment:
Do we want to backport this to 4.2?

Reply:
Yes, part of https://github.com/ravendb/ravendb/pull/11681